lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -2831,4 +2832,40 @@ describe('AwsCompileFunctions #2', () => {
       });
     });
   });
+
+  describe('function config', () => {
+    it('should not create a new version object if only function-wide configuration changed', () => {
+      return fixtures.setup('function').then(({ servicePath, updateConfig }) =>
```

Comment:
Can we upgrade this to async/await syntax?

Reply:
I was really hoping to do that! Yay!